src/style.css

```
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+input {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+}
+
+input.searchbar-small {
+  background: url(images/Search.svg) no-repeat;
+  background-size: 11px 11px;
+  background-position: 12px 14px;
```

yo have some duplicates
```suggestion
input {
  background: url(images/Search.svg) no-repeat;
  background-size: 19px 19px;
  background-position: 25px 24px;
}

input.searchbar-small {
  background-size: 11px 11px;
  background-position: 12px 14px;
```